lua/nvim-treesitter/textobjects/shared.lua

```diff
@@ -99,6 +116,8 @@ function M.textobject_at_point(query_string, pos, bufnr, opts)
     end
   elseif lookahead_largest_range then
     return bufnr, { lookahead_largest_range.node:range() }, lookahead_largest_range.node
+  elseif lookbehind_largest_range then
+    return bufnr, { lookbehind_largest_range.node:range() }, lookbehind_largest_range.node
   end
 end
 
```

Comment:
can you document your changes in the README?

Reply:
and sorry that it took me so long to review. I was quite busy at work (GTC arriving)

Reply:
No problem, i don't have much time either (school). Thank you for your time reviewing this pr and developing this plugin! It's great!